### PR TITLE
Add best practices chapter and fix some default values

### PR DESCRIPTION
**Pull request by amleczko**

### CODE REVIEW

File: crawlera.rst [hunk 1]
```diff
@@ -556,6 +552,18 @@ C#
 .. literalinclude:: _static/crawlera-proxy.cs
     :language: csharp
 
+Best practices
+==============
+We have prepared several best practices when using crawlera smart proxy.
+
+Set download timeout
+--------------------
+One of the most common problems our users have is too low download timeout in their scrapping application. Handling one request in crawlera can take a long time. This happens due to `throtling <http://doc.scrapinghub.com/crawlera.html#why-are-requests-slower-through-crawlera-as-opposed-to-using-proxies-directly>`_. Crawlera will try several times to process your request with different slaves and different delay time. Recommended value is **300s**. If you are using scrapy please check our `example configuration <http://doc.scrapinghub.com/crawlera.html#using-crawlera-with-scrapy>`_.
```
Comment (redapple) [inline crawlera.rst]: @amleczko , do you think `scrapy-crawlera` should change [its 1800 download_timeout value](https://github.com/scrapy-plugins/scrapy-crawlera/blob/master/scrapy_crawlera.py#L20)?

Comment (amleczko) [inline crawlera.rst]: 1800s is the maximum download time crawlera can handle and it is not recommended. 300s makes more sense from our experience.

Comment (oltarasenko) [inline crawlera.rst]: I want to double check wording here, e.g. `Crawlera will try several times to process your request with different slaves and different delay time` sounds incorrect to me, and I would suggest `Crawlera will try to process your request with different slaves and delay time`

Comment (oltarasenko) [inline crawlera.rst]: @amleczko

Comment (amleczko) [inline crawlera.rst]: Fixed.

**Pull request closed without merging**